packages/salesforcedx-utils/jest.config.js

```diff
@@ -1,8 +1,5 @@
 const baseConfig =  require('../../config/jest.base.config');
 
 module.exports = Object.assign({},
-  baseConfig,
-  {
-    testMatch: [ "**/unit/**/?(*.)+(spec|test).[t]s?(x)" ],
-  }
+  baseConfig
```

Comment:
clean up unnecessary override.

Reply:
👍🏽 
Also, TS is complaining about top line of this file for me:
<img width="1033" alt="2022-10-12_13-57-55" src="https://user-images.githubusercontent.com/46458081/195446606-af4944cf-95b6-453e-a299-a29346a9317c.png">

Reply:
cool..yeah it's a js file so can be ignored.  I can add a ignore comment I think

Reply:
just an FYI this was only showing u in the utils projects which using eslint vs tslint. So I only updated that config file